src/main/java/org/fundacionjala/coding/william/CamelCase.java

```diff
@@ -0,0 +1,27 @@
+package org.fundacionjala.coding.william;
+
+import java.util.StringJoiner;
+
+/**
+ * Class that converts the first letter of a word into a capital letter.
+ */
+public class CamelCase {
+
+    /**
+     * Method that converts the first letter of a word into a capital letter.
+     *
+     * @param word receives a string.
+     * @return returns a string.
+     */
+    public String camelCase(final String word) {
+        String[] array = word.split(" ");
+        StringJoiner result = new StringJoiner("");
```

Comment:
In this case is not necessary to use a StringJoiner, you can use directly StringBuilder